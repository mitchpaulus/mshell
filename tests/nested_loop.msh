0 outer-inc!
(
    "Outer: " @outer-inc str + wl
    0 inner-inc!
    (
        " Inner: " w @inner-inc str wl
        [(@inner-inc 2 >) (break)] if
        @inner-inc 1 + inner-inc!
    ) loop

    [(@outer-inc 2 >) (break)] if
    @outer-inc 1 + outer-inc!
) loop
