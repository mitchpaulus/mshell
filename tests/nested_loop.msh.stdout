Outer: 0
 Inner: 0
 Inner: 1
 Inner: 2
 Inner: 3
Outer: 1
 Inner: 0
 Inner: 1
 Inner: 2
 Inner: 3
Outer: 2
 Inner: 0
 Inner: 1
 Inner: 2
 Inner: 3
Outer: 3
 Inner: 0
 Inner: 1
 Inner: 2
 Inner: 3
